hack/e2e-suite/services.sh

```diff
@@ -252,7 +253,8 @@ function verify_from_container() {
             ok=false
             for j in $(seq -s' ' 1 10); do
               if wget -q -T 1 -O - http://$2:$3; then
-		ok=true
```

Comment:
Could you please fork these fixes into another PR? I'd like to get them in asap.